api/clients/eigenda_client_e2e_test.go

```diff
@@ -18,29 +19,71 @@ func init() {
 	flag.BoolVar(&runTestnetIntegrationTests, "testnet-integration", false, "Run testnet-based integration tests")
 }
 
+// TestClientUsingTestnet tests the eigenda client against holesky testnet disperser.
+// We don't test waiting for finality because that adds 12 minutes to the test, and is not necessary
+// because we already test for this in the unit tests using a mock disperser which is much faster.
 func TestClientUsingTestnet(t *testing.T) {
 	if !runTestnetIntegrationTests {
 		t.Skip("Skipping testnet integration test")
 	}
-	logger := log.NewLogger(log.NewTerminalHandler(os.Stderr, true))
-	client, err := clients.NewEigenDAClient(logger, clients.EigenDAClientConfig{
-		RPC:                      "disperser-holesky.eigenda.xyz:443",
-		StatusQueryTimeout:       25 * time.Minute,
-		StatusQueryRetryInterval: 5 * time.Second,
-		CustomQuorumIDs:          []uint{},
-		SignerPrivateKeyHex:      "2d23e142a9e86a9175b9dfa213f20ea01f6c1731e09fa6edf895f70fe279cbb1",
-		// Waiting for finality adds 12 minutes to the test, and is not necessary
-		// because we already test for this correct behavior in the unit tests using a mock disperser
-		// which is much faster.
-		WaitForFinalization: false,
+
+	t.Run("PutBlobWaitForConfirmationDepth0AndGetBlob", func(t *testing.T) {
+		t.Parallel()
+		logger := log.NewLogger(log.NewTerminalHandler(os.Stdout, true))
+		client, err := NewEigenDAClient(logger, EigenDAClientConfig{
+			RPC:                      "disperser-holesky.eigenda.xyz:443",
+			StatusQueryTimeout:       25 * time.Minute,
```

Comment:
should we comment with an equation on how to compute this depth.
Say batching interval is 3 mins, then attestation timeout is 80% of that. And writing to chain timeout is 72sec. So overall below 6 mins.

Reply:
wdym by attestation timeout? Is that the thing for 503? Haven't added that yet actually! So good reminder to add it, added comment in the 503 PR.

Why is your writing to chain timeout 72 sec here? Just random? It's only 3 confirmations, so should be 36 secs just waiting for ethereum, so you doubled that in case?

Reply:
Added comment in [7f7fdaf](https://github.com/Layr-Labs/eigenda/pull/821/commits/7f7fdaf885be3046c006002a1edb073248518827)
Not sure if it's what you had in mind..